src/app/tables/tests-table/tests-table.component.ts

```diff
@@ -65,6 +65,8 @@ export class TestsTableComponent implements OnChanges {
           'Test Case': result.testcase.name,
           'Logs': 'No logs available',
           'Waived': waivedTestCases.includes(result.testcase.name) ? 'Yes' : 'No',
+          'Impacts the decision': this.greenwaveDecision.satisfied_requirements.map(
+            function (j) { return j.result_id; }).includes(result.id) ? 'Yes' : 'No',
```

Comment:
Optional: It'd be nice to use the arrow function since that's the syntax we use everywhere else.